spec/features/spree/admin/taxjar_settings_spec.rb

```diff
@@ -15,16 +15,21 @@
       allow(ENV).to receive(:[]).and_call_original
       allow(ENV).to receive(:[]).with("TAXJAR_API_KEY").and_return(api_token)
       allow(SuperGood::SolidusTaxjar).to receive(:reporting_ui_enabled).and_return(true)
+    end
 
+    scenario "the user navigates the the TaxJar Settings" do
       visit "/admin"
       click_on "Settings"
       expect(page).to have_content("Taxes")
       click_on "Taxes"
       expect(page).to have_content("TaxJar Settings")
       click_on "TaxJar Settings"
+      expect(page).to have_content("TaxJar Settings")
```

Comment:
Might be better to have a different assertion here. The click could fail and this will still pass because that text is on the previous page.

Reply:
Added an assertion to check that the tab is active and for some basic content of the tab!